blocks/pocketguide/pocketguide.js

```diff
@@ -0,0 +1,23 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+
+/**
+ * loads and decorates the pocketguide
+ * @param {Element} block The pocketguide block element
+ */
+export default async function decorate(block) {
+  const cfg = readBlockConfig(block);
+  block.textContent = '';
+
+  // fetch footer content
+  const footerPath = cfg.footer || '/templates/pocketguide';
+  const resp = await fetch(`${footerPath}.plain.html`, window.location.pathname.endsWith('/templates/pocketguide') ? { cache: 'reload' } : {});
```

Comment:
I think the footer and footer cfg is not needed here, so you can go for the hard-coded template path for now.

Reply:
true, sorry didnt see this copy&waste part